Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rowanl22/firebase-tools:node10
```

Comment:
Can we move this under bipolar bridges instead?

Reply:
Done.